kuma/static/styles/minimalist/print.scss

```diff
@@ -0,0 +1,69 @@
+@import '../includes/vars';
+@import './mixins/print-mixins';
+
+@media print {
+    header {
+        display: none;
+    }
+
+    .center {
+        @include remove-center-spacing();
+    }
+
+    .notification-tray {
+        display: none;
```

Comment:
When you do it like this, the uglified output becomes:
```css
.notification-tray{display:none}.newsletter-box{display:none}.global-notice{display:none}.nav-footer{display:none} ...etc...
```
See [snapshot output](https://gist.github.com/peterbe/e686af33789999462485650075f3c9af)

The remedy would be to do something like this: https://github.com/peterbe/workon/blob/643ef0de2a3de09a0f1c460227b9c173ba91c36f/public/print.css#L1-L10

Reply:
Ah yes, I was not even thinking in that direction because I merely wanted to get everything into its own stylesheet, but this is *much* better.